### PR TITLE
Hindi TN support Cardinal, Decimal, Money, Time, Date, Fraction and Measure

### PR DESCRIPTION
**Pull request by ankitnv**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Have you signed your commits? Use ``git commit -s`` to sign.
- [ ] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [ ] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [ ] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [ ] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [ ] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [ ] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [ ] Remove import guards (`try import: ... except: ...`) if not already done.
- [ ] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [ ] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation
- [ ] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 *.model
```
Comment (mgrafu) [inline .gitignore:2]: please remove this file

File: nemo_text_processing/text_normalization/hi/clean_eval_data.py [hunk 2]
```diff
@@ -0,0 +1,342 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/clean_eval_data.py:1]: how does this file fit into the overall grammar?

File: nemo_text_processing/text_normalization/hi/data/numbers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/numbers/__init__.py:1]: can you make sure all new copyright files are dated 2024?

File: nemo_text_processing/text_normalization/hi/data/time/hours.tsv [hunk 4]
```diff
@@ -0,0 +1,24 @@
+१	एक
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/time/hours.tsv:1]: is this different from the numbers files?

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv [hunk 5]
```diff
@@ -0,0 +1,45 @@
+Hon.	Honorable
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv:1]: Since these are English, are they necessary for HI TN?

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv [hunk 6]
```diff
@@ -0,0 +1,14 @@
+st	street
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv:1]: Since this is English, is it necessary for HI TN?

File: nemo_text_processing/text_normalization/hi/data/whitelist/asr.tsv
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/asr.tsv:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv [hunk 7]
```diff
@@ -0,0 +1,71 @@
+Hon.	honorable
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/data/whitelist/ipa_symbols.tsv
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/ipa_symbols.tsv:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/data/whitelist/lj_speech.tsv [hunk 8]
```diff
@@ -0,0 +1,21 @@
+Mr.	mister
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/lj_speech.tsv:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/data/whitelist/symbol.tsv [hunk 9]
```diff
@@ -0,0 +1,23 @@
+&	and
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/data/whitelist/symbol.tsv:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/graph_utils.py [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+
+NEMO_CHAR = utf8.VALID_UTF8_CHAR
+
+NEMO_DIGIT = byte.DIGIT
+
+NEMO_HI_DIGIT = pynini.union("०", "१", "२", "३", "४", "५", "६", "७", "८", "९").optimize()
+NEMO_HI_NON_ZERO = pynini.union("१", "२", "३", "४", "५", "६", "७", "८", "९").optimize()
+NEMO_HI_ZERO = "०"
+
+NEMO_HEX = pynini.union(*string.hexdigits).optimize()
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/graph_utils.py:38]: are all the variables below unique to Hindi? If not, it would be best to have only the unique variables here and anything else imported from en/graph_utils

File: nemo_text_processing/text_normalization/hi/taggers/cardinal.py [hunk 11]
```diff
@@ -0,0 +1,324 @@
...
+        graph_ten_lakhs |= create_larger_number_graph(teens_and_ties, suffix_lakhs, 3, teens_ties)
+        graph_ten_lakhs |= create_larger_number_graph(teens_and_ties, suffix_lakhs, 2, graph_hundreds)
+        graph_ten_lakhs |= create_larger_number_graph(teens_and_ties, suffix_lakhs, 1, graph_thousands)
+        graph_ten_lakhs |= create_larger_number_graph(teens_and_ties, suffix_lakhs, 0, graph_ten_thousands)
+        graph_ten_lakhs.optimize()
+
+        # Crores graph ten crores graph
+        suffix_crores = pynutil.insert(" करोड़")
+        graph_crores = create_graph_suffix(digit, suffix_crores, 7)
+        graph_crores |= create_larger_number_graph(digit, suffix_crores, 6, digit)
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/taggers/cardinal.py:107]: since all function calls seem to be from X to 0, would it be possible to modify the function to call just once and have it internally do the iteration?

File: nemo_text_processing/text_normalization/hi/taggers/time.py [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+        # hour minute
+        graph_hm = self.hours + delete_colon + insert_space + self.minutes
+
+        final_graph = graph_hms | graph_hm
+
+        final_graph = self.add_tokens(final_graph)
+        self.fst = final_graph.optimize()
+
+
+# input_text = "१२:१०:१०"
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/taggers/time.py:61]: let's remove this code from all scripts

File: nemo_text_processing/text_normalization/hi/utils.py [hunk 13]
```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/hi/utils.py:2]: same comment as with graph_utils

File: nemo_text_processing/text_normalization/normalize.py [hunk 14]
```diff
@@ -41,7 +41,6 @@
 )
```
Comment (mgrafu) [inline nemo_text_processing/text_normalization/normalize.py:41]: let's make sure not to remove any code unnecessarily

File: nemo_text_processing/text_normalization/normalizeold.py
Comment (mgrafu) [inline nemo_text_processing/text_normalization/normalizeold.py:1]: let's remove this

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_date.txt [hunk 15]
```diff
@@ -0,0 +1,22 @@
+vierundzwanzigster juli zwei tausend dreizehn~24. Jul. 2013
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_date.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_decimal.txt [hunk 16]
```diff
@@ -0,0 +1,10 @@
+null komma zwei millionen~0,2 millionen
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_decimal.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_electronic.txt [hunk 17]
```diff
@@ -0,0 +1,9 @@
+c d f at a b c punkt e d u~cdf@abc.edu
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_electronic.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_fraction.txt [hunk 18]
```diff
@@ -0,0 +1,34 @@
+null nulltel~0/0
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_fraction.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_measure.txt [hunk 19]
```diff
@@ -0,0 +1,28 @@
+zwei hundert meter~200 m
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_measure.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_money.txt [hunk 20]
```diff
@@ -0,0 +1,23 @@
+zwei dollar~$2
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_money.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_ordinal.txt [hunk 21]
```diff
@@ -0,0 +1,20 @@
+ein hundertste~100.
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_ordinal.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_telephone.txt [hunk 22]
```diff
@@ -0,0 +1 @@
+null vier eins eins eins zwei drei vier eins zwei drei vier~(0411) 1234-1234
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_telephone.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_time.txt [hunk 23]
```diff
@@ -0,0 +1,24 @@
+acht uhr~8 Uhr
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_time.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_whitelist.txt [hunk 24]
```diff
@@ -0,0 +1,6 @@
+doktor dao~Dr. dao
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_whitelist.txt:1]: are these the correct test cases?

File: tests/nemo_text_processing/hi/test_cardinal.py [hunk 25]
```diff
@@ -0,0 +1,43 @@
...
+from nemo_text_processing.inverse_text_normalization.inverse_normalize import InverseNormalizer
+from nemo_text_processing.text_normalization.normalize import Normalizer
+from nemo_text_processing.text_normalization.normalize_with_audio import NormalizerWithAudio
+
+from ..utils import CACHE_DIR, RUN_AUDIO_BASED_TESTS, parse_test_case_file
+
+
+class TestCardinal:
+
+    #  inverse_normalizer = InverseNormalizer(lang='hi', cache_dir=CACHE_DIR, overwrite_cache=False)
```
Comment (mgrafu) [inline tests/nemo_text_processing/hi/test_cardinal.py:27]: let's remove commented out code from all scripts

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/en_tn_True_deterministic_lower_cased__tokenize.far
Comment (mgrafu) [inline tools/text_processing_deployment/hi_tn_grammars_lower_cased/en_tn_True_deterministic_lower_cased__tokenize.far:1]: is this necessary?

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far
Comment (mgrafu) [inline tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far:1]: is this necessary?

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/verbalize/verbalize.far
Comment (mgrafu) [inline tools/text_processing_deployment/hi_tn_grammars_lower_cased/verbalize/verbalize.far:1]: is this necessary?

File: nemo_text_processing/text_normalization/hi/clean_eval_data.py [hunk 26]
```diff
@@ -0,0 +1,342 @@
...
+from nemo_text_processing.text_normalization.data_loader_utils import (
+    EOS_TYPE,
+    Instance,
+    load_files,
+    training_data_to_sentences,
+)
+
+
+"""
+This file is for evaluation purposes.
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/hi/clean_eval_data.py:28]: If this is for evaluation it should be removed from the PR. We want as much unity across languages as possible.

File: nemo_text_processing/text_normalization/hi/data/numbers/thousands.tsv [hunk 27]
```diff
@@ -0,0 +1,8 @@
+१०००	हज़ार
```
Comment (tbartley94) [inline nemo_text_processing/text_normalization/hi/data/numbers/thousands.tsv:1]: I'm confused what this data file is for. The trailing 0s are supposed to be managed by your cardinal class. This should just have matching between the thousands place digit.

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv (hunk 5 above)
Comment (tbartley94) [reply nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv:1]: There's likely a fair amount of codeswitching even in native Hindi text. So this may be valid. However, probably the cleaner way is to stick with only Hindi and have a separate PR for codeswitched text.

File: .gitignore (hunk 1 above)
Comment (ankitnv) [reply .gitignore:2]: Done

File: nemo_text_processing/text_normalization/hi/clean_eval_data.py (hunk 2 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/clean_eval_data.py:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/numbers/__init__.py (hunk 3 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/numbers/__init__.py:1]: Done

File: nemo_text_processing/text_normalization/hi/data/time/hours.tsv (hunk 4 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/time/hours.tsv:1]: In Hindi, we typically use a 12-hour clock format when referring to time. To handle this properly, I created separate files for time instead of using the numbers TSV. I have also defined a specific TSV file exclusively for time.

File: nemo_text_processing/text_normalization/hi/data/time/seconds.tsv [hunk 28]
```diff
@@ -0,0 +1,60 @@
+०१	एक
```
[the remark this answers is not in the document]
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/time/seconds.tsv:1]: In Hindi, we typically use a 12-hour clock format when referring to time. To handle this properly, I created separate files for time instead of using the numbers TSV. I have also defined a specific TSV file exclusively for time.

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv (hunk 5 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv (hunk 6 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/asr.tsv
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/asr.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv (hunk 7 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/ipa_symbols.tsv
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/ipa_symbols.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/lj_speech.tsv (hunk 8 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/lj_speech.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/data/whitelist/symbol.tsv (hunk 9 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/data/whitelist/symbol.tsv:1]: I’m unsure how this file fits into the overall grammar. Should it be retained, or would it be better to remove it?

File: nemo_text_processing/text_normalization/hi/taggers/time.py (hunk 12 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/taggers/time.py:61]: Done

File: nemo_text_processing/text_normalization/normalizeold.py
Comment (ankitnv) [reply nemo_text_processing/text_normalization/normalizeold.py:1]: Done

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_date.txt (hunk 15 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_date.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_cardinal.txt [hunk 29]
```diff
@@ -0,0 +1,753 @@
+
```
[the remark this answers is not in the document]
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_cardinal.txt:1]: Done

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_decimal.txt (hunk 16 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_decimal.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_electronic.txt (hunk 17 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_electronic.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_fraction.txt (hunk 18 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_fraction.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_money.txt (hunk 20 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_money.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_telephone.txt (hunk 22 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_telephone.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_ordinal.txt (hunk 21 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_ordinal.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_time.txt (hunk 23 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_time.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_whitelist.txt (hunk 24 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/data_inverse_text_normalization/test_cases_whitelist.txt:1]: No, Remove It

File: tests/nemo_text_processing/hi/test_cardinal.py (hunk 25 above)
Comment (ankitnv) [reply tests/nemo_text_processing/hi/test_cardinal.py:27]: Done

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far
Comment (ankitnv) [reply tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far:1]: No, Remove It

File: typescript
[the remark this answers is not in the document]
Comment (ankitnv) [reply typescript:1]: No, Remove It

File: nemo_text_processing/text_normalization/hi/clean_eval_data.py (hunk 2 above)
Comment (mgrafu) [reply nemo_text_processing/text_normalization/hi/clean_eval_data.py:1]: if you're unsure, maybe try removing it and running tests. if everything passes without it, commit and push the deletion

File: .gitignore (hunk 1 above)
Comment (mgrafu) [reply .gitignore:2]: can you push the commit with the changes you made?

File: nemo_text_processing/text_normalization/hi/data/time/hours.tsv (hunk 4 above)
Comment (mgrafu) [reply nemo_text_processing/text_normalization/hi/data/time/hours.tsv:1]: can you check if English creates a separate tsv? if not, let's follow the logic in English, which also uses 12-hour clock format

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv (hunk 5 above)
Comment (mgrafu) [reply nemo_text_processing/text_normalization/hi/data/whitelist/alternatives.tsv:1]: echoing Travis's suggestion about separate PR for code-switched TN. Let's keep only Hindi content. Check if the file is used in your whitelist grammar. If not, delete; if it is, change for Hindi terms only

File: nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv (hunk 6 above)
Comment (mgrafu) [reply nemo_text_processing/text_normalization/hi/data/whitelist/alternatives_all_format.tsv:1]: same suggestion as for the above file

File: nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv (hunk 7 above)
Comment (mgrafu) [reply nemo_text_processing/text_normalization/hi/data/whitelist/asr_with_pc.tsv:1]: same suggestion for all whitelist files that contain words in English

File: nemo_text_processing/text_normalization/hi/clean_eval_data.py (hunk 2 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/hi/clean_eval_data.py]: Done

File: .gitignore (hunk 1 above)
Comment (ankitnv) [reply .gitignore]: I have committed all the changes and successfully pushed them.

**Comment by mgrafu** [comment]
please merge main to this PR so all other languages are up to date for testing

File: .gitignore (hunk 1 above)
Comment (mgrafu) [reply .gitignore]: this file is still there, please delete

**Comment by ankitnv** [comment]
> please merge main to this PR so all other languages are up to date for testing

Yes the main branch is merged

File: nemo_text_processing/text_normalization/en/data/money/per_unit.tsv
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/data/money/per_unit.tsv:1]: why were these deleted?

File: nemo_text_processing/text_normalization/normalize.py [hunk 30]
```diff
@@ -144,9 +139,6 @@ def __init__(
         elif lang == 'es':
             from nemo_text_processing.text_normalization.es.taggers.tokenize_and_classify import ClassifyFst
             from nemo_text_processing.text_normalization.es.verbalizers.verbalize_final import VerbalizeFinalFst
-        elif lang == 'fr':
-            from nemo_text_processing.text_normalization.fr.taggers.tokenize_and_classify import ClassifyFst
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/normalize.py:148]: why?

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far
Comment (ekmb) [inline tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far:1]: remove .far files

**Review by ekmb — CHANGES_REQUESTED** [review]
Please remove .far files from the PR and don't change anything related to other languages

File: tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far
Comment (ankitnv) [reply tools/text_processing_deployment/hi_tn_grammars_lower_cased/classify/tokenize_and_classify.far:1]: Yes, I'll remove this filr.

File: nemo_text_processing/text_normalization/en/data/money/per_unit.tsv
Comment (ankitnv) [reply nemo_text_processing/text_normalization/en/data/money/per_unit.tsv:1]: Sorry, it was deleted by mistake I'll make sure to accomadate incoming and current changes,

File: .gitignore
Comment (ankitnv) [inline .gitignore:1]: I'll make sure to ignore this file while next push

File: Jenkinsfile
Comment (ankitnv) [inline Jenkinsfile:1]: I'll make sure to update Jenkins file as per the main branch

File: typescript
Comment (ankitnv) [inline typescript:1]: I'll remove this file as well with next push

File: nemo_text_processing/text_normalization/en/data/money/per_unit.tsv
Comment (ankitnv) [reply nemo_text_processing/text_normalization/en/data/money/per_unit.tsv:1]: I have reverted the file

File: nemo_text_processing/text_normalization/normalize.py (hunk 30 above)
Comment (ankitnv) [reply nemo_text_processing/text_normalization/normalize.py:148]: reverted

**Review by ankitnv — COMMENTED** [review]
updated requested changes

**Review by ankitnv — COMMENTED** [review]
changes updated
